build/Dockerfile

```diff
@@ -70,7 +70,10 @@ COPY --from=download /download/kubernetes/node/bin/kube-proxy.exe .
 
 # Copy CNI plugin binaries
 WORKDIR /payload/cni-plugins/
-COPY --from=download /download/cni-plugins/* .
```

Comment:
Won't `COPY --from=download /download/cni-plugins/ .` do the trick? Or are you trying to be explicit so that in the future, we can only copy the files that we actually need?

Reply:
That's correct.